lib/matplotlib/mathtext.py

```diff
@@ -407,51 +407,48 @@ def destroy(self):
 
     def get_kern(self, font1, fontclass1, sym1, fontsize1,
                  font2, fontclass2, sym2, fontsize2, dpi):
-        r"""
+        """
         Get the kerning distance for font between *sym1* and *sym2*.
 
-        *fontX*: one of the TeX font names::
-
-          tt, it, rm, cal, sf, bf or default/regular (non-math)
-
-        *fontclassX*: TODO
-
-        *symX*: a symbol in raw TeX form. e.g., '1', 'x' or '\sigma'
-
-        *fontsizeX*: the fontsize in points
-
-        *dpi*: the current dots-per-inch
+        See `get_metrics` for a detailed description of the parameters.
         """
         return 0.
 
     def get_metrics(self, font, font_class, sym, fontsize, dpi, math=True):
         r"""
-        *font*: one of the TeX font names::
-
-          tt, it, rm, cal, sf, bf or default/regular (non-math)
-
-        *font_class*: TODO
-
-        *sym*:  a symbol in raw TeX form. e.g., '1', 'x' or '\sigma'
-
-        *fontsize*: font size in points
-
-        *dpi*: current dots-per-inch
-
-        *math*: whether sym is a math character
-
-        Returns an object with the following attributes:
-
-        - *advance*: The advance distance (in points) of the glyph.
-
-        - *height*: The height of the glyph in points.
-
-        - *width*: The width of the glyph in points.
-
-        - *xmin*, *xmax*, *ymin*, *ymax* - the ink rectangle of the glyph
+        Parameters
+        ----------
+        font : str
+            One of the TeX font names: tt, it, rm, cal, sf, bf, default/regular
+            (non-math), bb, frak, scr.
```

Comment:
Thechnically, I we should put quotes around the names.

Reply:
I intentionally did not, the names are so short that adding quotes makes punctuation more than 50% of the characters which does not help readability.

Reply:
IMHO in particular single quotes wouldn't be too obtrusive, but I won't fight over that.

What I don't understand is the "default/regular (non-math)" case. Would that be `get_metrics('default/regular', ...)`?